haas/plugins/parallel_runner.py

```diff
@@ -46,9 +46,9 @@ def __call__(self, result):
         self.results.append(result)
 
 
-def _run_test_in_process(test_case):
+def _run_test_in_process(test_case, buffer):
     result_handler = ChildResultHandler()
-    result_collector = ResultCollector(buffer=True)
```

Comment:
Note, that I don't fully understand the usage of `ResultCollector`, and in particular why its instantiation happens inside `_run_test_in_process()`. This is a simple attempt at solving issue #182.

Reply:
The currentl implementation is very similar to setting `buffer=False` which results in output from the test going to the parent sys.stdout and sys.stderr. So the rest of the changes are not necessary. 

Yet, not buffering the test output is not really a correct solution to the issue #182. The main problem is that test executed in parallel will write into the parent stdout/stderr streams at the same time making the results unreadable. 

The correct solution is to buffer the output for each test and print it out in the parent process when the result has been collected. This way the output for each test will be selfcontained and readable. 

Unfortunately, at the moment while both buffer stdout and stderr streams are created (i.e.`buffer=True`). When the results are accumulated, the stdout/stderr content is ignored unless there was a test exception/failure (see https://github.com/scalative/haas/blob/master/haas/result.py#L249). I think that fixing #182 will invlove fixing the behaviour (and maybe design) of the TestResult class.